manifests/0000_50_olm_02-services.yaml

```diff
@@ -17,6 +17,10 @@ spec:
       port: 8081
       protocol: TCP
       targetPort: metrics
+    - name: https-pprof
+      port: 8443
+      protocol: TCP
+      targetPort: 8443
```

Comment:
Unfortunately, we'll likely need to land #107 before we can directly modify the root list of manifests. You would need to update the staging/operator-lifecycle-manager helm chart templates to have these changes propagate to this directory. In the script responsible for generating the list of manifests, we always nuke the root manifests directly and replace it with a tmp directory that contained the rendered helm chart templates. The verification check will fail as `make manifests && git diff` will produce a diff as this PR introduced manifests, and updates existing manifests, which will removed due to the process outlined above.

Reply:
And ideally, we wouldn't be making direct modifications to the staging/* directory paths as we're likely heading down the path of adding validation that ensures changes made to the staging directories are only from cherrypick-ed upstream commits.

Reply:
Changes to manifests are now integrated into the `make manifests` command. No files in the `staging/*` directory were hurt as a part of this change.